modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-lvm-thin-volumes_{context}"]
+= About LVM thin volumes
+
+To use advanced storage features such as creating volume snapshots or volume cloning, you must provision a logical volume manager (LVM) thin-pool on the {op-system-ostree} host and configure both the logical volume manager storage (LVMS) provider and the cluster. LVM thin-pools must be attached to a volume group.
+
+[IMPORTANT]
+====
+To create CSI snapshots, you must configure thin volumes on the {op-system-ostree} host. The CSI does not support volume shrinking because of the potential for data loss.
```

Comment:
I think shrinking is not related to snapshots?

Reply:
Once you expand your storage with thin volumes, the CSI isn't going to shrink them. It's a caveat to going this route. From what I read, the lack of support from the CSI is related to volume restoring. If you shrink the volumes, then try to restore, you lose data.